main.test.js

```diff
@@ -0,0 +1,60 @@
+const { expect, describe } = require("@jest/globals");
```

Comment:
- [ ] Kindly update the test for the count of all items based on the new implementation and please make sure that your test includes all edge cases like if there are 0 items on the page.